web/modules/custom/bnf/tests/src/Unit/Mapper/NodeArticleMapperTest.php

```diff
@@ -71,29 +76,31 @@ public function testNodeArticleMapping(): void {
       canonicalUrl: CanonicalLink::make(
         url: 'https://example.dk'
       ),
-      overrideAuthor: 'this is an author',
-      showOverrideAuthor: TRUE,
+      // This will be overwritten by the mapper.
+      overrideAuthor: NULL,
+      // This will be overwritten by the mapper.
+      showOverrideAuthor: FALSE,
       subtitle: 'this is the subtitle',
-      teaserImage: NULL,
       teaserText: 'this is a teaser text',
-      paragraphs: [
-        ParagraphTextBody::make(
-          id: '982e0d87-f6b8-4b84-8de8-c8c8bcfef999',
-          body: Text::make(
-            format: 'with_format', value: 'This is the text')
-        ),
-      ]
     );
 
     $node = $mapper->map($graphqlArticle);
 
     $this->assertSame($node, $this->entityProphecy->reveal());
+    $this->entityProphecy->set('status', Node::PUBLISHED)->shouldHaveBeenCalled();
     $this->entityProphecy->set('title', 'this is the title')->shouldHaveBeenCalled();
     $this->entityProphecy->set('field_subtitle', 'this is the subtitle')->shouldHaveBeenCalled();
-    $this->entityProphecy->set('field_override_author', 'this is an author')->shouldHaveBeenCalled();
-    $this->entityProphecy->set('field_show_override_author', TRUE)->shouldHaveBeenCalled();
     $this->entityProphecy->set('field_publication_date', ["value" => "2025-01-01"])->shouldHaveBeenCalled();
     $this->entityProphecy->set('field_teaser_text', 'this is a teaser text')->shouldHaveBeenCalled();
+    $this->entityProphecy->set('field_teaser_image', [])->shouldHaveBeenCalled();
+    $this->entityProphecy->set('field_canonical_url', ["uri" => "https://example.dk"])->shouldHaveBeenCalled();
+    $this->entityProphecy->set('field_paragraphs', [])->shouldHaveBeenCalled();
+
+    // Testing that the mapper always sets this to true.
```

Comment:
Ah, but you're missing a test: that it import the override author if there's one on the origin node.